iothub/service/src/RegistryManager.cs

```diff
@@ -46,6 +54,62 @@ public static RegistryManager CreateFromConnectionString(string connectionString
             return new HttpRegistryManager(iotHubConnectionString, transportSettings);
         }
 
+#if !NET451
+
+        /// <summary>
+        /// Creates an instance of <see cref="RegistryManager"/>.
```

Comment:
This generates a self-reference, meaning the link in the help page will take you to the same page you are on. I don't recommend adding these links.